pkgs/by-name/ew/eww/package.nix

```diff
@@ -17,13 +19,25 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = "elkowar";
     repo = "eww";
-    rev = "refs/tags/v${version}";
+    rev = "v${version}";
     hash = "sha256-rzDnplFJNiHe+XbxbhZMEhPJMiJsmdVqtZxlxhzzpTk=";
   };
 
+  patches = [
+    # FIXME: remove with the next update
```

Comment:
Would be best to also explain the patch's purpose, or at least link to the issue

Reply:
Done.